backends/platform/atari/readme.txt

```diff
@@ -0,0 +1,422 @@
+ScummVM
+=======
+
+This is a new port of ScummVM (https://www.scummvm.org), a program which allows
+you to run certain classic graphical adventure and role-playing games, provided
+you already have their data files.
+
+You can find a full list with details on which games are supported and how well
+on the compatibility page: https://www.scummvm.org/compatibility.
+
+
+Yet another port?
+-----------------
+
+Yes, I am aware of the official Atari/FreeMiNT port done by KeithS over the
+years (https://docs.scummvm.org/en/v2.6.1/other_platforms/atari.html). It is
+even updated every release and put on the official ScummVM website. That port
+is basically just a recompiled SDL backend for our platform - that has some
+advantages (works in GEM, can be easily compiled for the FireBee etc) but also
+a few disadvantages, like:
+
+- It's huge, massively huge, about 51 MiB. It contains about every engine,
+  every library which is available to link on our platform. Most of that stuff
+  is not needed, see my next point.
+
+- A good half of the features are basically useless on our platform (like
```

Comment:
Because there's a limited horsepower available on our platform, features like hi-res 16bpp graphics, software synthesizers, scalers, real-time software  MP3/OGG/FLAC playback etc., are omitted. This saves memory and disk space, making the whole port more lightweight.